### PR TITLE
Adding Antoine CAILLOT

### PR DESCRIPTION
**Pull request by caillotantoine**
Added my member's page and information.

### CODE REVIEW

File: _data/members.yml [hunk 1]
```diff
@@ -672,7 +672,8 @@
   family: Osorio
   role: phd
   from: 2022-05-01
-  nationality: cr
+  nationality: crwebsite: "https://antoineandre.github.io/"
```
Comment (aescande) [inline _data/members.yml:675]: I think you put some data at the wrong place. Anyway, there is a mistake here.

File: _data/members.yml [hunk 2]
```diff
@@ -672,7 +672,8 @@
...
+  scholar: "https://scholar.google.fr/citations?user=iopU-RcAAAAJ&hl"
```
Comment (aescande) [inline _data/members.yml:676]: This is also misplaced

File: en/members/member-acaillot.html [hunk 3]
```diff
@@ -0,0 +1,5 @@
+---
+layout: default
+title: CNRS-AIST JRL
+---
+{% include member_full.html id="jrlmember" %}
```
Comment (fkanehiro) [inline en/members/member-acaillot.html:5]: jrlmember->acaillot

**Comment by caillotantoine** [comment]
I corrected the highlighted comments. May you check the corrections please.

**Review by gergondet — APPROVED** [review]

**Pull request merged**